getting-started/queries.md

```diff
@@ -159,5 +159,63 @@ For more information about these functions, see the API documentation for
 
 </Procedure>
 
-[first]: /api/:currentVersion:/hyperfunctions/first
-[last]: /api/:currentVersion:/hyperfunctions/last
+## Use time buckets to get values
+
+To make it easier to look at numbers over different time ranges, you can use the
+Timescale `time_bucket` function. Time buckets are used to group data, so that
+you can perform calculations over different time periods.
+
+In this section, you use the same query as the previous section to find the
+`first` and `last` values, but start by organizing the data into 1-hour time
+buckets.
```

Comment:
`first` and `last` aggregations return the first and last seen value for the current time bucket grouping and sorting accordingly.